presto-main/src/main/java/com/facebook/presto/SystemSessionProperties.java

```diff
@@ -1661,7 +1661,7 @@ public SystemSessionProperties(
                 booleanProperty(
                         REWRITE_CONSTANT_ARRAY_CONTAINS_TO_IN_EXPRESSION,
                         "Rewrite contant array contains to IN expression",
-                        true,
+                        false,
```

Comment:
Session properties should always be backed by configuration properties and the default value should be based on a corresponding configuration property.

Reply:
Done.